penman/types.py

```diff
@@ -2,14 +2,15 @@
 Basic types used by various Penman modules.
 """
 
-from typing import Any, Iterable, List, Tuple, Union
+from typing import Iterable, List, Tuple, Union
 
 Variable = str
 Constant = Union[str, float, int, None]  # None for missing values
 Role = str  # '' for anonymous relations
+Symbol = str
 
 # Tree types
-Branch = Tuple[Role, Any]
+Branch = Tuple[Role, Union[Symbol, "Node"]]
 Node = Tuple[Variable, List[Branch]]
```

Comment:
Is it possible to have an AMR tree that consists of just a single `Variable` with no branches?

Reply:
Probably not for AMR, but it is a possible graph for this Penman library:

```pycon
>>> penman.parse('(a)')  # tree node has empty branch list
Tree(('a', []))
>>> penman.decode('(a)').triples  # graph has :instance None
[('a', ':instance', None)]
```